StigData/Archive/DotNet/U_MS_DotNet_Framework_4-0_STIG_V1R8_Manual-xccdf.log

```diff
@@ -0,0 +1,5 @@
+V-81495::Use regedit to review the following Windows registry keys::Value: 1
+V-81495::For 32-bit systems: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\.NETFramework\v4.0.30319\::Registry Hive: HKEY_LOCAL_MACHINE
+V-81495::For 64 bit systems:::Registry Path: \SOFTWARE\Microsoft\.NETFramework\v4.0.30319\
+V-81495::HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\.NETFramework\v4.0.30319\::Type: REG_DWORD
+V-81495::If the “SchUseStrongCrypto” value name does not exist, or is not a REG_DWORD type set to “1”, this is a finding.::Value Name: SchUseStrongCrypto 
```

Comment:
The previous version of this STIG has logfile entries for rule V-30935.  Are those no longer valid in the latest version of the STIG?

Reply:
running Compare-PowerStigXml: 
#TYPE System.Collections.DictionaryEntry
"Name","Key","Value"
"V-18395","V-18395","changed"
"V-30935","V-30935","deleted"
"V-32025","V-32025","deleted"

I believe they no longer apply

Reply:
oh I see Nehru's comment below, on the compare results. I'll sync with him